### ISSUE TITLE
License WebView scroll positive lost after device orientation change

### ISSUE THREAD

**Issue by littlecd** [TeamNewPipe/NewPipe#10431]
### Checklist

- [X] I am able to reproduce the bug with the latest version given here: [CLICK THIS LINK](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.25.2

### Steps to reproduce the bug

1. Go to About and view licenses
2. scroll up
3. rotate screen

### Expected behavior

Scroll positive should be change

### Actual behavior

The page scrolls to top

### Screenshots/Screen recordings

Before rotation
<img width="310" alt="Screen Shot 2023-09-19 at 2 33 22 PM" src="https://github.com/TeamNewPipe/NewPipe/assets/144590216/dd9886a8-0180-4c4a-9872-b71ff633cbae">

After rotation
<img width="609" alt="Screen Shot 2023-09-19 at 2 33 32 PM" src="https://github.com/TeamNewPipe/NewPipe/assets/144590216/ee904c9d-95a5-465f-a4d7-8e2d2aa7234c">

### Affected Android/Custom ROM version

Android 10

### Affected device model

emulator

**Comment by TobiGr** [comment]
I don't think that there is going to be an easy fix for this. The AlertDialog and the WebView are recreated on rotation, the old ones are removed from the view. Creating a complex logic to retrieve the nearest paragraph or something like that seems too much effort for something unimportant like a license view.
